smapply/tasks.py

```diff
@@ -20,13 +22,11 @@ def sync_all_users():
 
         profile = Profile.objects.filter(smapply_id=sma_user['id']).first()
         if not profile:
-            serializer = UserSerializer(data=sma_user)
-            if serializer.is_valid():
-                user = process_user(serializer.data)
-                user.profile.smapply_user_data = sma_user
-                user.profile.save()
-
+            try:
+                user = process_user(sma_user)
                 profiles_to_sync.append(user.profile)
+            except:  # pylint: disable=bare-except
+                log.exception('Syncing profile for SMA user %s failed', sma_user['id'])
```

Comment:
As soon as I started bootcamp up I got some of these exceptions, caused by users in smapply without last names.  Not sure if that's a good thing (we were missing errors we should have known about before) or a bad thing (these errors are expected and should be ignored)?

```ValidationError: {'last_name': [ErrorDetail(string='This field may not be blank.', code='blank')]}```

Reply:
These errors will keep popping up whenever `smapply.tasks.sync_all_users` is called (default every 15 minutes) unless/until all the users are fixed in smapply.

Reply:
works without logging exceptions now, @pdpinch should we be notified if some users are invalid when processed via `sync_all_users`?  On the dev smapply account there are users without last names which cause the validation errors.

Reply:
Anna and I discussed this and I think it's OK as is. We care more about this users once they start the application process.

Reply:
@mbertrand,I spoke with Peter about this, we are okay not getting notified at this stage, hoping that we are going to get an error when processing the webhook for that user, if the user decides to create an application.